mlrun/runtimes/function.py

```diff
@@ -952,6 +952,17 @@ def invoke(
             data = json.loads(data)
         return data
 
+    def _trigger_of_kind_exists(self, kind: str) -> bool:
+        if not self.spec.config:
+            return False
+
+        for name, spec in self.spec.config.items():
+            if isinstance(spec, dict):
+                if spec.get("kind") == kind:
+                    return True
```

Comment:
What if the `spec` is a dict but does not have a `kind`?
e.g for something other than triggers

we don't want to fail in this case, then:
```
if spec.get("kind", None) == kind:
```

Reply:
@TomerShor None is default value for [`get`](https://www.tutorialspoint.com/python/dictionary_get.htm) function, so we don't need to set it explicitly